Clarify #6.1 statements (#173)
openid/oid4vc-haip: openid4vc-high-assurance-interoperability-profile-1_0.md

## ISO mdoc specific requirements for OpenID for Verifiable Presentations over W3C Digital Credentials API

Requirements for both the Wallet and the Verifier:

> * ISO mdoc Credential Format specific DCQL parameter, `intent_to_retain` defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be present.

Reviewer: ```suggestion

* ISO mdoc Credential Format specific DCQL parameter, `intent_to_retain` defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be present.
```
needed for rendering

> * ISO mdoc Credential Format specific DCQL parameters, intent_to_retain and doctype as defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be present.

Reviewer 2: Since `doctype` is now required in OpenID4VP i don't think the requirement is necessary anymore (and added backticks to intent_to_retain)

```suggestion
* ISO mdoc Credential Format specific DCQL parameter, `intent_to_retain` defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be present.
```
